docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -11,9 +11,9 @@ The :ref:`scheduler` creates, starts, stops, and destroys each :ref:`container`
 of your app. For example, a command such as ``deis scale web=3`` tells the
 scheduler to run three containers from the Docker image for your app.
 
-Deis defaults to using the :ref:`fleet_scheduler`. Tech previews of schedulers based on
-:ref:`Kubernetes <k8s_scheduler>`, :ref:`Mesos with Marathon <mesos_scheduler>`, and
-:ref:`Swarm <swarm_scheduler>` are available for testing.
+Deis defaults to using the :ref:`fleet_scheduler`, but the :ref:`Kubernetes <k8s_scheduler>`
```

Comment:
"the Kubernetes" should be changed to "the Kubernetes scheduler" so it rolls off the tongue better :)